openedx/core/djangoapps/xblock/runtime/runtime.py

```diff
@@ -387,14 +399,11 @@ class XBlockRuntimeSystem:
     class can be used with many different XBlocks, whereas each XBlock gets its
     own instance of XBlockRuntime.
     """
-    STUDENT_DATA_EPHEMERAL = 'ephemeral'
-    STUDENT_DATA_PERSISTED = 'persisted'
-
     def __init__(
         self,
-        handler_url,  # type: Callable[[UsageKey, str, Union[int, ANONYMOUS_USER]], str]
-        student_data_mode,  # type: Union[STUDENT_DATA_EPHEMERAL, STUDENT_DATA_PERSISTED]
-        runtime_class,  # type: XBlockRuntime
+        handler_url: Callable[[UsageKey, str, UserType | None], str],
+        student_data_mode: StudentDataMode,
+        runtime_class: XBlockRuntime,
```

Comment:
```suggestion
        runtime_class: type[XBlockRuntime],
```

We expect a class, not an instance.